augur/filter.py

```diff
@@ -105,6 +105,8 @@ def register_arguments(parser):
                                 help="Exclude samples matching these conditions. Ex: \"host=rat\" or \"host!=rat\". Multiple values are processed as OR (matching any of those specified will be excluded), not AND")
     parser.add_argument('--include-where', nargs='+',
                                 help="Include samples with these values. ex: host=rat. Multiple values are processed as OR (having any of those specified will be included), not AND. This rule is applied last and ensures any sequences matching these rules will be included.")
+    parser.add_argument('--exclude-ambiguous-dates', choices=['all', 'days', 'months'],
+                                help='Exclude ambiguous dates Ex: days - excludes 2020-09-XX, months - exclueds 2020-xx-19, all - excludes any ambiguous dates')
```

Comment:
This help text looks generally great. Can you fix the typo of "exclueds" to "excludes"?